src/liboslexec/oslexec_pvt.h

```diff
@@ -2081,6 +2083,8 @@ class OSLEXECPUBLIC ShadingContext {
 
     Dictionary *m_dictionary;
 
+    OCIOColorSystem m_ocio_system;
```

Comment:
Just checking... is any CUDA_ARCH needed around this?

Reply:
Builds fine for me.